optimum/intel/openvino/configuration.py

```diff
@@ -473,8 +505,13 @@ def __init__(
         self.compression = kwargs.get(
             "compression", None
         )  # A field for backward-compatability of training-time compression parameters
-        bits = self.quantization_config.bits if self.quantization_config else None
-        self.dtype = "int" + str(bits) if isinstance(bits, int) else dtype
+        if self.quantization_config is not None:
+            if isinstance(self.quantization_config, OVWeightQuantizationConfig):
+                self.dtype = self.quantization_config.weight_format
+            else:
+                self.dtype = f"int{quantization_config.bits}"
```

Comment:
```suggestion
                self.dtype = "int8"
```

Reply:
suggestion from https://github.com/huggingface/optimum-intel/pull/882#discussion_r1740986547 shouldn't have be applied in the first place